package.json

```diff
@@ -67,7 +67,7 @@
     "serve": "^10.1.2",
     "stylelint": "^9.10.1",
     "stylelint-scss": "^3.9.2",
-    "uswds": "^2.0.3",
+    "uswds": "^2.7.0",
```

Comment:
This looks like a big change, should we be worried?

Reply:
@zachmargolis Yeah, we should check what has been updated between these two versions - I have a Jira ticket to research this update.